controllers/authController.js

```diff
@@ -168,6 +168,7 @@ const logout = (req, res) => {
 const changePassword = async (req, res) => {
   try {
     const { email, currentPassword, newPassword } = req.body;
+    console.log("test")
 
     if (!email || !currentPassword || !newPassword) {
       return res.status(400).json({ message: "Please provide all fields" });
```

Comment:
{
    "message": "User validation failed: role: Path `role` is required."
}

Revisit the mongo syntax to only update what needs to be changed. try to look for $set.